### PR TITLE
Feat/user network calls

### PR DESCRIPTION
**Pull request by ryanwholey**
User network requests ENGAGE. :dancers:

### CODE REVIEW

**Comment by ryanwholey** [comment]
closes #25

File: server.js [hunk 1]
```diff
@@ -19,16 +19,17 @@ app.use(function(req, res, next) {
...
+    console.log('delete',req.params);
+    res.json({success:true, userId:req.params.id});
   })
   .put(function(req, res){
-    res.send();
-  });
+    console.log('put',req.body);
+    // respond with updated user object here
+    // from db? better options? 
+    res.json({mock:'DB NOT SETUP YET',userId: req.params.id});
```
Comment (danthareja) [inline server.js]: what about a little https://github.com/typicode/json-server until the back end is totally set up?

File: client/services/network.jsx [hunk 2]
```diff
@@ -39,6 +33,13 @@ const getCharacters = () => {
 };
 
 
+const userRequests = (method, userID, rawParams={}) => {
+  const params = makeParams(method, rawParams);
+  var a = `${url}/users/${userID}`
+  return fetch(`${url}/users/${userID}`, params)
```
Comment (danthareja) [inline client/services/network.jsx]: any reason to declare `a` here?

Comment (ryanwholey) [inline client/services/network.jsx]: @danthareja oop! nope that should/will be removed. :+1:

File: server.js (hunk 1 above)
Comment (ryanwholey) [inline server.js]: @danthareja OOH! this is so neat!!

Comment (naomiajacobs) [inline server.js]: 0.0

**Comment by ryanwholey** [comment]
@danthareja json-server added, pretty sweet!!

**Pull request merged**